### PR TITLE
feat: selecting profiles

### PR DESCRIPTION
**Pull request by borkopetrovicc**
## Description

https://www.figma.com/file/qLl8KxBOOr640h18ICx0H6/Web-app?type=design&node-id=2344-62791&mode=design&t=Ph6Hfz6Pr7AtyUbw-0

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: src/components/ProfileBox/AddNewProfile.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const AddNewProfile = ({ onClick }: Props) => {
           />
         </svg>
       </div>
-      <h2 className={styles.title}>Add new</h2>
+      <h2 className={styles.title}>Add profile</h2>
```
Comment (ChristiaanScheermeijer) [inline src/components/ProfileBox/AddNewProfile.tsx]: Translations via i18next?

File: src/components/ProfileBox/ProfileBox.module.scss [hunk 2]
```diff
@@ -49,9 +49,8 @@
         justify-content: center;
         align-items: center;        
         // border: 1px solid rgba(255, 255, 255, 0.3);
```
Comment (ChristiaanScheermeijer) [inline src/components/ProfileBox/ProfileBox.module.scss]: You can remove this commented code

File: src/components/ProfileBox/ProfileBox.module.scss [hunk 3]
```diff
@@ -49,9 +49,8 @@
...
-        background: #1F232C;
-        border: 1px solid transparent;
-        border-radius: 100px;
+        border: 1px solid white;
+        border-radius: 75px;
```
Comment (ChristiaanScheermeijer) [inline src/components/ProfileBox/ProfileBox.module.scss]: You could create an SCSS variable (on top of this file) for the avatar size used for multiple properties.

```scss
$avatar-size: 75px;

...
```

**Review by ChristiaanScheermeijer — COMMENTED** [review]
I left a few comments, I see that none of the texts are translated. Is this on your to-do list?

**Comment by borkopetrovicc** [comment]
> I left a few comments, I see that none of the texts are translated. Is this on your to-do list?

Yes it is on my to-do list.

File: src/components/ProfileBox/AddNewProfile.tsx (hunk 1 above)
Comment (borkopetrovicc) [reply src/components/ProfileBox/AddNewProfile.tsx]: db138a252ab26d75a6f649041fff24356614ca05

File: src/components/ProfileBox/ProfileBox.module.scss (hunk 2 above)
Comment (borkopetrovicc) [reply src/components/ProfileBox/ProfileBox.module.scss]: db138a252ab26d75a6f649041fff24356614ca05

File: src/components/ProfileBox/ProfileBox.module.scss (hunk 3 above)
Comment (borkopetrovicc) [reply src/components/ProfileBox/ProfileBox.module.scss]: db138a252ab26d75a6f649041fff24356614ca05

**Comment by budzworthy** [comment]
> > I left a few comments, I see that none of the texts are translated. Is this on your to-do list?
> 
> Yes it is on my to-do list.

You can create a PR in draft status if you are still finishing up things like this, and then reviewers won't automatically get notified.

**Pull request merged**